### DISCUSSION TITLE
Consider introducing Long Term Support releases

### DISCUSSION THREAD

**Discussion by thomas-bc** [nasa/fprime#2628] [Call for Comments]
| | |
|:---|:---|
|**_F´ Version_**| |
|**_Affected Component_**|  |
---
## Feature Description

We may want to consider introducing the notion of [Long Term Support releases](https://en.wikipedia.org/wiki/Long-term_support)

## Rationale

More stable and usable software for users.

**Comment by LeStarch** [comment]
If we were to do LTS support, we'd need to switch to formal release branches.  Once we do that, then hot-fixes, security fixes, etc would be doable in the context of extended support. 

We'd also need to enumerate exactly what "support" consists of.  My instinct is: bug fixes, and security fixes.  Feature development would not be supported.

@timcanham, @SterlingPeet, @arizvi786  in your experience ...would LTS releases of F´ help what you do?  If so, what things would you need to be long-term supported and what would not be supported (as a differentiation from just updating from devel).

**Comment by Joshua-Anderson** [comment]
The issue I see is that many projects want to be using the latest version of fprime to get the latest features, until they reach the point in development that they need to freeze changes. Given the current rapid development of fprime, I'm not sure many projects would take us up on the LTS offer. 

On the JPL side of things, I think the answer might be that we're happy to create/maintain a fprime LTS release for a specific project that's willing to pay for the maintenance costs, and we'd do it publicly so other projects could then take advantage of that LTS release as well.
